content/index.fr.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Vue d'ensemble des standards de l'accessibilité"
+permalink: /fundamentals/fr
+ref: /fundamentals/
+lang: fr
+last_updated: 2020-12-28
+translators:
+- name : "Sofia Ahmed"
+contributors:
+- name : "Sandra Velarde Gonzalez (ETNIC)"
+github:
+  repository: w3c/wai-fundamentals-overview
+  path: content/index.fr.md
+footer: > # Text in footer in HTML
+  <p><strong>Date :</strong> Mise à jour : 19 août 2020.</p>
+  <p><strong>Rédactrice :</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Développé par le Groupe de travail Éducation et Promotion (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+
+---
+
+Les ressources suivantes contiennent des introductions à l'accessibilité :
```

Comment:
The English is "information for getting started with accessibility". The draft translation should be more close to that, rather than "introductions". Dear French speakers: Suggestions? Thanks!